src/noCommentedOutCodeRule.ts

```diff
@@ -72,6 +75,10 @@ class NoCommentedOutCodeRuleWalker extends ErrorTolerantWalker {
     private textIsTslintFlag(text: string): boolean {
         return text.startsWith('tslint:');
     }
+    
+    private textIsNote(text: string): boolean {
+        return /^(NOTE|TODO|FIXME|BUG|HACK|XXX)[:( ]/.test(text);
```

Comment:
`[:( ]`

With the `(`, I see you're trying to handle this case:

> FIXME(bobby): foo

Would you be able to make it check for both `(` and `)` followed by the `:`? Thoughts?

Valid:

> FIXME: foo
> FIXME(bobby): foo

Invalid:
> FIXME(bobby) foo
> FIXME(bobby: foo

Of course, make sure to add all of these cases as tests.